virttest/qemu_vm.py

```diff
@@ -1828,6 +1828,18 @@ def sort_key(dev):
                               pci_bus=pci_bus)
                 iov += 1
 
+        #  Add general PCI Pass-through devices
+        for pci_pass_devs in params.objects("pci_pass_devices"):
+            pci_pass_params = params.object_params(pci_pass_devs)
+            # required
```

Comment:
Let us have better comment message.

Reply:
As you have explained about what is required and what is optional params, it can be commented to make it complete about what the params is. It will help everyone to understand your implementation better.